private/stamp.bash

```diff
@@ -1,20 +1,15 @@
-#!/bin/bash
-
-set -o errexit
-set -o nounset
-set -o pipefail
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
 
 # Do not error if the user doesn't have gcloud installed.
 if [ -z "${PROJECT_ID:-}" ]; then
   if which gcloud > /dev/null; then
     PROJECT_ID="$(gcloud config get-value core/project)"
   else
-    PROJECT_ID="NO-PROJECT"
+    PROJECT_ID="no-project"
```

Comment:
why this change?

Reply:
this should error out as well since it's a generic name. while uppercase chars lead to cryptic messages in some registries, this will show meaningful error message like `no such project no-project`. I am fine reverting it though.

Reply:
kk lets leave it like this.